.github/scripts/unittest-linux/run_test.sh

```diff
@@ -34,5 +34,5 @@ fi
     export TORCHAUDIO_TEST_ALLOW_SKIP_IF_NO_MOD_inflect=true
     export TORCHAUDIO_TEST_ALLOW_SKIP_IF_NO_MOD_pytorch_lightning=true
     cd test
-    pytest torchaudio_unittest -k "not backend and not /io/ and not prototype and not ffmpeg and not fairseq and not hdemucs and not (torchscript and rnnt) and not torchscript_consistency"
+    pytest torchaudio_unittest -k "not torchscript"
```

Comment:
Locally, I can confirm that the following pytest call
```
pytest test/torchaudio_unittest -k "(not torchscript) and (not (lfilter and cuda)) and not (no_lm_decoder or construct_basic_decoder_2 or construct_basic_decoder_3 or construct_basic_decoder_6 or construct_basic_decoder_7 or custom_kenlm_decoder)"
```
is 100% successful.

The decoder errors are due to missing kenlm support in flashlight-text (building kenlm is broken when using Python 3.13, although, there is https://github.com/kpu/kenlm/pull/473).

So, I suspect that the following
```sh
pytest test/torchaudio_unittest -k "(not torchscript) and (not (lfilter and cuda))
```
is a minimal pytest call provided that all dependencies are installed. Apparently, not all optional dependencies (fairseq, ...) are not available in CI, hence the CI failures.

For now, I suggest to use
```suggestion
    pytest torchaudio_unittest -k "not torchscript and not fairseq and not demucs" 
```
(I did not suggest `(not (lfilter and cuda))` because apparently CI has not issues with the corresponding test.)